transport/dlr-malspp/transport/src/main/java/de/dlr/gsoc/mo/malspp/transport/SPPEndpoint.java

```diff
@@ -365,7 +365,7 @@ public void sendMessage(final MALMessage msg)
         try {
             if (isLocalDestination) {
                 try {
-                    ((SPPMessageBody) msg.getBody()).prepareInProcessBody();
+                    //((SPPMessageBody) msg.getBody()).prepareInProcessBody();
```

Comment:
Remove the line entirely. I am big opponent of commenting code out in git commits usually.